global.json

```diff
@@ -3,12 +3,12 @@
     "vs": {
       "version": "15.9"
     },
-    "dotnet": "6.0.100-preview.4.21255.9"
+    "dotnet": "6.0.100-preview.5.21302.13"
   },
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "6.0.0-beta.21357.3"
   },
   "sdk": {
-    "version": "6.0.100-preview.4.21255.9"
+    "version": "6.0.100-preview.5.21302.13"
```

Comment:
this will need to be updated to at least 6.0.100-preview.7.21364.4 to get the compiler version that supports file-scoped namespaces.

Reply:
```suggestion
    "version": "6.0.100-preview.7.21364.30"
```